ddtrace/profiling/collector/_memalloc_tb.c

```diff
@@ -18,6 +18,53 @@ static PyObject* empty_string = NULL;
 
 #define TRACEBACK_SIZE(NFRAME) (sizeof(traceback_t) + sizeof(frame_t) * (NFRAME - 1))
 
+static PyObject* ddframe_class = NULL;
+
+bool
+memalloc_ddframe_class_init()
+{
+    if (ddframe_class) {
+        Py_DECREF(ddframe_class);
+        ddframe_class = NULL;
+    }
+
+    PyObject* collections = PyImport_ImportModule("collections");
+    if (collections == NULL) {
+        PyErr_Print();
+        return false;
+    }
+
+    PyObject* namedtuple_fun = PyObject_GetAttrString(collections, "namedtuple");
+    Py_DECREF(collections);
+    if (namedtuple_fun == NULL) {
+        PyErr_Print();
+        return false;
+    }
+
+    // Very important that this matches the definition of DDFrame in event.py
+    PyObject* class_name = PyUnicode_FromString("DDFrame");
+    PyObject* field_names = Py_BuildValue("(ssss)", "file_name", "lineno", "function_name", "class_name");
```

Comment:
Perhaps worth checking that we get valid Python objects here?

Reply:
Got rid of this entirely; now it's pulling from event.py